boards/cc2650stk/Makefile.include

```diff
@@ -0,0 +1,24 @@
+export CPU        = cc26x0
+export CPU_MODEL ?= cc26x0f128
+
+#export PORT      ?= /dev/ttyUSB0
+#include $(RIOTBOARD)/Makefile.include.serial
+
+#export DEBUGGER = $(RIOTBOARD)/$(BOARD)/dist/debug.sh
+#export DEBUGSERVER = JLinkGDBServer -device CC2538SF53
+#export RESET = $(RIOTBOARD)/$(BOARD)/dist/reset.sh
+
+export UNIFLASH_PATH ?= missing
```

Comment:
@haukepetersen i've removed the line (less is more ;-)
this happens:
```
Flash program /uniflash.sh not found. Aborting.
```
i find that reasonable